src/main/kotlin/org/jitsi/nlj/transform/node/Node.kt

```diff
@@ -178,7 +178,7 @@ sealed class StatsKeepingNode(name: String) : Node(name) {
     /**
      * Keeps stats for this [Node]
      */
-    private val stats = NodeStats()
+    protected val stats = NodeStats()
```

Comment:
Why do you need to do this?  Is it just so that `PacketLossNode` can use `stats.numInputPackets` rather than tracking its own `packetsSeen` as the old `PacketLoss` node did?

Reply:
Yes, that's the only reaso

Reply:
I brought back the (two lines of) code that keeps track of the number of packets